ggshield/scan/docker.py

```diff
@@ -347,12 +326,24 @@ def docker_scan_archive(
             ignored_detectors=ignored_detectors,
             ignore_known_secrets=ignore_known_secrets,
         )
-        task_scan = progress.add_task(
-            "[green]Scanning Docker Image...", total=len(files.files)
-        )
-        results = scanner.scan(
-            files.files,
-            progress_callback=partial(progress.update, task_scan),
-        )
+        with create_progress_bar(doc_type="files") as progress:
+            task_scan = progress.add_task("[green]Scanning Docker config...", total=1)
+            results = scanner.scan(
+                [docker_image.config_scannable],
+                progress_callback=partial(progress.update, task_scan),
+            )
+
+        for info, files in docker_image.get_layers():
+            display_info(f"Scanning layer {info.command}")
```

Comment:
Yes, I wanted to avoid giant lines, but I agree it's not clear right now. Maybe we need some `display_heading()` function which would use a different color, or just print a blank line before each "Scanning layer (...)" line.

Going to experiment with this.

Reply:
Again, that's a nice to have IMO.